include/utils/CommonUtils.php

```diff
@@ -2699,6 +2699,27 @@ function getUItypeByFieldName($module, $fieldname) {
 	return $uitype;
 }
 
+/**
+ * Function to get the Type of data for a field by the fieldname.
+ * Takes the input as $module - module name,and fieldname of the field
+ * returns the type of data, varchar type
+ */
+function getTypeOfDataByFieldName($module, $fieldname) {
+	global $log, $adb;
+	$log->debug('> getTypeOfDataByFieldName ' . $module);
+	$tabIdList = array();
+	//To find tabid for this module
+	$tabIdList[] = getTabid($module);
+	if ($module == 'Calendar') {
```

Comment:
We do not need this anymore. Calendar does not exist anymore and if it ever does again it will be a normal module. You can use a unique value for tabid instead of an array which will be faster.